actions/get-vault-secrets/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: Get Vault Secrets
+description: Composite action (step) to get secrets from Grafana Labs' Vault instance.
+inputs:
+  repo_secrets:
+    description: |
+      Repository-specific secret mapping. Only Grafana Labs employees can list these secrets in Vault.
+      These secrets are stored in the `ci/data/repo/<repo owner>/<repo name>/<secret path>` path in Vault.
+      Ex:
+      ```
+      repo_secrets: |
+        ENVVAR1=secretpath:key
+        ENVVAR2=secretpath:key2
+      ```
+  common_secrets:
+    description: |
+      Common secrets mapping. Only Grafana Labs employees can list these secrets in Vault.
+      These secrets are stored in the `ci/data/common/<secret path>` path in Vault.
+      Ex:
+      ```
+      repo_secrets: |
+        ENVVAR1=secretpath:key
+        ENVVAR2=secretpath:key2
+      ```
+
+runs: 
+  using: composite
+  steps:
+    - id: vault-iap-auth
+      uses: google-github-actions/auth@v1
+      with:
+        # Note that these aren't secrets, login is secured through Github's OIDC integrations with GCP and Vault.
+        # Get with: 
+        #   gcloud iam workload-identity-pools list --project=grafanalabs-workload-identity --location="global"
+        #   gcloud iam workload-identity-pools providers list --workload-identity-pool=<POOL>
+        workload_identity_provider: "projects/304398677251/locations/global/workloadIdentityPools/github/providers/github-provider"
```

Comment:
maybe these would be good as org-wide _variables_ (not secrets)

https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/

Reply:
`vars` isn't usable in composite actions either